.devops/deploy-pipelines.yml

```diff
@@ -40,15 +40,33 @@ parameters:
   - name: 'OPENAPI_SPEC_PATHS'
     type: object
     default:
-      api_auth: 'api_auth.yaml'
-      api_backend: 'api_backend.yaml'
-      api_bonus: 'api_bonus.yaml'
-      api_bpd: 'api_bpd.yaml'
-      api_myportal: 'api_myportal.yaml'
-      api_notifications: 'api_notifications.yaml'
-      api_pagopa: 'api_pagopa.yaml'
-      api_public: 'api_public.yaml'
-      api_session: 'api_session.yaml'
+      api_auth: 
```

Comment:
maybe it's better using consistent prefixes like `@pagopa/io-sdk-backend-auth`
(from the wider scope to the tighter)

Reply:
> maybe it's better using consistent prefixes like `@pagopa/io-sdk-backend-auth`
> (from the wider scope to the tighter)

I tried to reuse the patter of the code gen, that adds _"-sdk"_ at the end of the package name

Reply:
The pattern `io-sdk-<api_name>` makes me think it's somehow related to https://github.com/pagopa/io-sdk, that's why I would avoid it.

> (from the wider scope to the tighter)

That's actually what we thought: 
```
scope of api (application name) --> scope of package ("-sdk")
```
Anyway, so far we aimed to not have to name each package, but being able to infer the name from the application which is serving the api (see https://github.com/pagopa/io-utils/blob/master/src/commands/gen-api-sdk/index.ts#L65). We're still in time to change this for a better pattern, if any proposal exists.

This is not the case of the current application, which exposes several different apis from the application. Hence, we need to name each one.

Reply:
Any thoughts? Can we proceed with those names?